services/builder/configProcessor.go

```diff
@@ -63,14 +63,20 @@ func configProcessor(c pb.ContextBoxServiceClient, wg *sync.WaitGroup) error {
 			defer wg.Done()
 		}
 
-		// check if Desired state is null and if so we want to delete the existing cluster
+		// check if Desired state is null and if so we want to delete the existing config
 		if config.DsChecksum == nil && config.CsChecksum != nil {
 			if err := destroyConfig(config, c); err != nil {
 				log.Error().Err(err).Send()
 			}
 			return
 		}
 
+		// check for cluster deleting
+		configToDelete := prepareConfigToDelete(config)
+		if err := destroy(configToDelete, c); err != nil {
+			log.Error().Err(err).Send()
```

Comment:
I wonder why we are changing `log.Error().Msgf()` to `log.Error().Err().Send`?

Afaik, the `Msgf()` already calls `Send()`, plus it gives us ability to format the message as we please.

From [docs](https://pkg.go.dev/github.com/rs/zerolog@v1.28.0#Event.Msgf):
```
Msgf sends the event with formatted msg added as the message field if not empty.
```

Reply:
I've used `log.Error().Err().Send` in #280, Although I think we should stay uniform across all service and just stick to either `log.Error().Msgf()` or `log.Error().Err().Send` personally since we mostly use the former I'd stick to it

Reply:
I am personally fine with any approach but we need to stick to one for a consistent pattern.
@Despire

Reply:
Since we mostly use `log.Error().Msgf()`, it would make more sense to stick to it, as @Despire said.

@jaskeerat789, please make appropriate changes in your PR, we will deal with the rest later.